.sample.env

```diff
@@ -0,0 +1,11 @@
+COUPON_CREATOR_ADDR=
+ETHERSCAN_API_KEY=
+BINANCESCAN_API_KEY=
+TRUFFLE_MNEMONIC=
+INFURA_KEY=
+DEBUG_CONTRACT_VERIFICATION=true
+DAO_NAME=
```

Comment:
This is missing from the list in the `README`. Should it be included with a description?

Reply:
Fixed. Thanks for flagging.